rules/manager.go

```diff
@@ -115,19 +115,24 @@ type Rule interface {
 	Eval(context.Context, time.Time, *promql.Engine, *url.URL) (promql.Vector, error)
 	// String returns a human-readable string representation of the rule.
 	String() string
+
+	UpdateEvalTimeSeconds(float64)
```

Comment:
Below we have `GetEvaluationTimeSeconds` and `setEvaluationTimeSeconds` while here the names are different for same functionality. Can we sync those?

Reply:
Got it. Thanks for the review! 👍